SQL/Archive/17.1/2017-03-10-addUserAccountsRoles.sql

```diff
@@ -0,0 +1,29 @@
+ALTER TABLE `permissions` DROP FOREIGN KEY `fk_permissions_1`;
```

Comment:
this should be moved to /SQL/Archive/2017-03-10-addUserAccountsRoles.sql

Reply:
@taracampbell this has not yet been done

Reply:
@ridz1208 Done